src/components/data-entry/QueryItem/Action.tsx

```diff
@@ -8,7 +8,7 @@ export interface IActionProps {
   onClick?: () => void
 }
 
-export const Action = (props: IActionProps) => {
+const Action = (props: IActionProps) => {
   let buttonClassNames: string = 'query-item query-item--action'
   if ((props.type ?? 'default') === 'default') buttonClassNames += ` query-item--secondary`
   if ((props.type ?? 'default') === 'disabled') buttonClassNames += ` query-item--disabled`
```

Comment:
I know it's not from this PR but a quick suggestion if you think it's worth it.

```suggestion
  const { type = 'default' } = props.type
  if (type === 'default') buttonClassNames += ` query-item--secondary`
  if (type === 'disabled') buttonClassNames += ` query-item--disabled`
```